### DISCUSSION TITLE
Separation of Sound and Light modes.

### DISCUSSION THREAD

**Discussion by InterplanetaryGentlemann** [gpstar81/GPStar-proton-pack#292] [Ideas]
Feature Request for the ability to mix and match the sound and light sequences from the different films. As someone looking to make more of a hybrid pack, I would love to be able to run the '84/'89 sound effects while still using the Afterlife Cyclotron effect. If it is possible to make it so the light and sound sequences from the films are separate EEPROM settings to be changed around, that would be wonderful.

**Comment by gpstar81** [comment]
Hello, yes it is something that will be looked into very shortly.

**Comment by nomakewan** [comment]
This is now being tracked in issue #355 and is slated for the v6.0.0 firmware.
